app/views/user_mailer/delete_work_notification.text.erb

```diff
@@ -1,9 +1,12 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
-  
-Your work "<%= @work.title.html_safe %>" was deleted at <%= (@work.pseuds.count > 1 && @user != User.current_user) ? "the request of " + text_pseud(User.current_user.default_pseud) : "your request" %>.
-  
-If you have questions, please contact Support at <%= root_url %>support.
-  
-Attached is a copy of your work for your reference.
-<% end %>
\ No newline at end of file
+<%= t('.hello', user:  @user.default_pseud.byline) %>
+<%= (@work.pseuds.count > 1 && @user != User.current_user) ?
+t('.part1_other',
+        title: @work.title,
+        pseud: User.current_user.default_pseud) :
+t('.part1_yourself',
+        title: @work.title) %>
+<%= t('.part2', support: t('.support')+' '+new_feedback_report_url) %>
```

Comment:
Agree with sarken's comments about adding spaces so it's easier to parse. I'm also unsure about this _translated support text + a space + a URL_ hard-coded syntax as I think that will limit the ability of a translator to change the ordering if for some reason the URL needs to be before or within the text. (It might also be that given how often we use it, the "add a link to the Support form" functionality could be a helper function somewhere to aid readability and consistency, but let's address that later)

Do we also want to standardise on double quotes while we're doing this?